scripts/setup-shared-content.sql

```diff
@@ -0,0 +1,42 @@
+-- For more information, refer to https://docs.snowflake.com/en/developer-guide/native-apps/preparing-data-content
+
+-- mark that our application package depends on an external database in
+-- the provider account. By granting "reference_usage", the proprietary data
+-- in the chairlift_provider_data database can be shared through the app
+grant reference_usage on database chairlift_provider_data
```

Comment:
It might be worth adding a note here:
1. If using snowCLI, then it will use package owner role to execute this script. 
2. If not using snowCLI, do you need to explicitly add in the `use role` statement?

Reply:
this will only run with snowCLI. I can add a comment to differentiate this from provider/create-package.sql file.